tribute-page/index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Tribute Page Dr. Norman Borlaug</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <main id="main" class="main-container">
+    <h1 id="title">Dr. Norman Borlaug</h1>
+    <p>The man who saved a billion lives</p>
+    <figure id="img-div" class=" image-container">
+      <img id="image" class="main-image"
+        src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+        alt="Dr. Norman Borlaug seen standing in Mexican wheat field with a group of biologists" />
+      <figcaption id="img-caption" class="img-description">
+        Dr. Norman Borlaug, third from the left, trains biologists in Mexico
+        on how to increase wheat yields - part of his life-long war on hunger.
+      </figcaption>
+    </figure>
+    <section id="tribute-info" class="bio-graphy">
+      <h3 id="topic" class="subject">Here's a time line of Dr. Borlaug's life:</h3>
+      <ul>
```

Comment:
You should avoid using strong tags, again and again, they do not come in symentic HTML tags. Apply the same class to ul li tags and apply the strong property to it from the CSS file.

Reply:
- [ ] Kindly make sure you rephrase your comments and make them in a friendly and polite way.
- [ ] Also, always explain why the changes are suggested :smile: